static/styles/main.css

```diff
@@ -0,0 +1,174 @@
+/* ---------------------------------------------------------- GoogLe Fonts ---------------------------------------------------------- */
+@import url('https://fonts.googleapis.com/css2?family=Oleo+Script+Swash+Caps:wght@400;700&family=Poppins:ital,wght@0,300;0,500;1,600&display=swap');
+/* 
+font-family: 'Oleo Script Swash Caps', cursive;
+font-family: 'Poppins', sans-serif;
```

Comment:
Please notice that the comments are here again

Reply:
Just kicking them out.